packages/grafana-ui/src/components/IconButton/IconButton.story.tsx

```diff
@@ -35,28 +35,86 @@ const meta: Meta<typeof IconButton> = {
   },
 };
 
-export const Examples = () => {
+export const Basic: StoryFn<typeof IconButton> = (args: IconButtonProps) => {
+  return <IconButton {...args} />;
+};
+
+export const ExamplesSizes = () => {
+  const sizes: IconSize[] = ['xs', 'sm', 'md', 'lg', 'xl'];
+  const icons: IconName[] = ['search', 'trash-alt', 'arrow-left', 'times'];
+  const variants: IconButtonVariant[] = ['primary', 'secondary', 'destructive'];
+
   return (
-    <div>
-      <RenderScenario background="canvas" />
-      <RenderScenario background="primary" />
-      <RenderScenario background="secondary" />
+    <div
+      className={css`
+        button {
+          margin-right: 8px;
+          margin-left: 8px;
+          margin-bottom: 20px;
+        }
+      `}
+    >
+      <VerticalGroup spacing="md">
```

Comment:
Could we make this horizontal? In my screen it cuts off the top title 😞 
<img width="267" alt="Screenshot 2023-05-12 at 13 46 47" src="https://github.com/grafana/grafana/assets/100691367/4bb080d3-9c5a-420b-a48c-ca500f40785a">